CHANGELOG.md

```diff
@@ -2,6 +2,22 @@
 
 All notable changes to this project will be documented in this file.
 
+## [0.16.0] - 2017-01-26
+### Added
+- Add Kubernetes probe
+- Retrieve Open vSwitch port metrics
+- Allow traffic capture on Open vSwitch ports
+- Add host info, such as CPU, memory, OS, Open vSwitch options, to metadata
+- Add `Preferences` pane to the Web UI
+- Allow SSH to agents through the WebUI (thanks to Dede)
+
```

Comment:
- [build] MacOS Windows client binary

Reply:
I suggest we put this for the next release when we'll make the binary available for download